Wrappers/Python/cil/plugins/tigre/Geometry.py

```diff
@@ -0,0 +1,171 @@
+from cil.framework import AcquisitionGeometry, ImageGeometry
+import numpy as np
+
+try:
+    from tigre.utilities.geometry import Geometry
+except ModuleNotFoundError:
+    raise ModuleNotFoundError("This plugin requires the additional package TIGRE\n" +
+            "Please install it via conda as tigre from the ccpi channel")
+
+class CIL2TIGREGeometry(object):
```

Comment:
I'm wondering if in the longer term, we shouldn't modify this class so that it also creates the astra geometry?